### ISSUE TITLE
Options in Settingsmenu

### ISSUE THREAD

**Issue by TobiasF92** [TobiasF92/FS22_interactiveControl#12]
I would like to collect ideas for the settings menu, so every one can set up IC for their own uses.

IC Settings:

- Deactivate IC while leaving vehicle ->[true/false]

Feel free to answer with your ideas :)

**Comment by HatsuneKid** [comment]
This proposal is to avoid some players making trouble in multiplayer games to activate the IC of others' vehicles?

**Comment by TobiasF92** [comment]
no, my proposal is just to de/activate a reset of the current state (Clickpoint visible or not)

Players from other farms should already not be able to operate your IC

**Comment by HatsuneKid** [comment]
all right. I think some players will need it.

**Comment by AgrarKadabra** [comment]
A "hotbutton" to temporarily enable the IC Hud would be nice. 
In other words, only show the IC hud while holding down a key (for e.g. spacebar).

**Comment by atledreier** [comment]
An option to keep IC enabled, but hide the icons would be amazing. Maybe combined with the hotkey functionality, where a hotkey temporarily shows the icons if you're searching for one you don't use all the time.
